Resources/Prototypes/Catalog/VendingMachines/Inventories/medidrobe.yml

```
@@ -1,20 +1,23 @@
-- type: vendingMachineInventory
-  id: MediDrobeInventory
-  startingInventory:
-    ClothingBackpackDuffelMedical: 4
-    ClothingBackpackMedical: 4
-    ClothingBackpackSatchelMedical: 4
-    ClothingUniformJumpsuitParamedic: 4
-    ClothingUniformJumpskirtParamedic: 4
-    ClothingHeadNurseHat: 4
-    ClothingUniformJumpskirtNurse: 4
-    ClothingOuterCoatLab: 4
-    ClothingShoesColorWhite: 4
-    ClothingHandsGlovesLatex: 4
-    ClothingHeadsetMedical: 4
-    ClothingOuterWinterMed: 2
-    ClothingOuterWinterPara: 2
-    ClothingOuterCoatParamedicWB: 2
-    ClothingHeadHatParamedicsoft: 2
-    ClothingHeadHelmetVoidParamed: 1
-    ClothingOuterHardsuitVoidParamed: 1
+- type: vendingMachineInventory
+  id: MediDrobeInventory
+  startingInventory:
+    ClothingBackpackDuffelMedical: 4
+    ClothingBackpackMedical: 4
+    ClothingBackpackSatchelMedical: 4
+    ClothingUniformJumpsuitParamedic: 4
+    ClothingUniformJumpskirtParamedic: 4
+    ClothingHeadNurseHat: 4

```

Was missing in the new diff
```suggestion
    ClothingHeadNurseHat: 4
    ClothingUniformJumpskirtNurse: 4
```